numpy/core/src/npymath/npy_math_complex.c.src

```diff
@@ -31,9 +34,11 @@
 #include "npy_math_common.h"
 #include "npy_math_private.h"
 
-/*==========================================================
- * Custom implementation of missing complex C99 functions
- *=========================================================*/
+#define raise_inexact() do { volatile npy_float junk = 1 + tiny; } while(0)
```

Comment:
I was always temped to remove this entirely.  The inexact exception is not exposed to python AFAICT.